### PR TITLE
Added Tool Tips to buttons in music and video player

### PR DESCRIPTION
**Pull request by AlexandreGaubil**
- [ ] This change has been discussed with the author.
- [x] It implements / fixes issue #2871.

---

**Description:**

Adds Tool Tips to buttons in the music player (eg, Show Album Art, Go back to video mode, etc.), video player (eg, Quick Settings, Picture-in-Picture, etc.) & playlist selector (Repeat, Shuffle, Add, Remove, etc.).
Partially fixes the issue #2871 (more precisely, the second feature request).

### CODE REVIEW

File: iina/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -260,6 +260,16 @@
 "osc_toolbar.music_mode" = "Enter music mode";
 "osc_toolbar.sub_track" = "Choose subtitle track";
 
+"mini_player.album_art" = "Toggle Album Art";
```
Comment (saagarjha) [inline iina/Base.lproj/Localizable.strings:263]: I'd lean towards making this lowercase, unless you think there's a good reason why it should be capitalized?

File: iina/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -260,6 +260,16 @@
 "osc_toolbar.music_mode" = "Enter music mode";
 "osc_toolbar.sub_track" = "Choose subtitle track";
 
+"mini_player.album_art" = "Toggle Album Art";
```
Comment (saagarjha) [inline iina/en.lproj/Localizable.strings:263]: Same here.

**Comment by AlexandreGaubil** [comment]
@saagarjha Absolutely, I hadn't payed attention to that. I just fixed it.

**Comment by alejx** [comment]
Haven’t tried it but looks reasonable to me. Will try it later.

**Comment by lhc70000** [comment]
Merged locally via eeaf17a.

**Pull request closed without merging**